rpcs3/Emu/RSX/VK/VKHelpers.cpp

```diff
@@ -569,4 +569,17 @@ namespace vk
 
 		return false;
 	}
+
+	const std::string get_version()
+	{
+		uint32_t instance_version = VK_API_VERSION_1_0;
+
+		vkEnumerateInstanceVersion(&instance_version);
+
+		const uint32_t vulkan_major = VK_VERSION_MAJOR(instance_version);
+		const uint32_t vulkan_minor = VK_VERSION_MINOR(instance_version);
+		const uint32_t vulkan_patch = VK_VERSION_PATCH(VK_HEADER_VERSION);
+
+		return fmt::format("%d.%d.%d", vulkan_major, vulkan_minor, vulkan_patch);
```

Comment:
This is wrong. Dont mix SDK header version with instance version - they are not the same thing.
`vulkan_patch = VK_VERSION_PATCH(instance_version)` is the correct value here (header vesion is always hardcoded to current khronos version so its actually 1.1.73 for example). You should log it separately because in future we might decide to dynamically link vulkan1.
```
Vulkan Instance Version: Bleh
Vulkan SDK Revision: X
```

Reply:
As discussed on Discord, I applied the required changes.